pandas/tests/test_frame.py

```diff
@@ -10771,6 +10771,13 @@ def test_diff(self):
         assert_series_equal(the_diff['A'],
                             tf['A'] - tf['A'].shift(1))
 
+        # issue 10907
+        df = pd.DataFrame({'x': pd.Series([1]),'y': pd.Series([2]), 'z': pd.Series([3])})
```

Comment:
this is doesn't actually test anything (and it works), but it doesn't replicate the original issue problem. IIRC you had that their, but it has changed.